### ISSUE TITLE
No support for Ubuntu 22.10?

### ISSUE THREAD

**Issue by jdrch** [bit-team/backintime#1368]
I just updated to Ubuntu 22.10. When I run `# apt update`, I get this:

```
Err:24 https://ppa.launchpadcontent.net/bit-team/stable/ubuntu kinetic Release
  404  Not Found [IP: 185.125.190.52 443]
```

Any ideas?

**Comment by buhtz** [comment]
Thanks for reporting. That is correct.

The maintaining team is new but the @Germar is also still with us. It is not yet decided how we treat the PPA. We need to discuss that in the team.

You should use the [Ubuntu repo](https://help.ubuntu.com/community/Repositories/Ubuntu). But keep in mind that Back In Time is only available via "Ubuntu universe" which is even not official Ubuntu but only kind of a community version.

I would suggest two things
 1. Open a ticket at Ubuntu to update the current "Ubuntu universe" package.
 2. Open a ticket to make Canonical officially supoort Back In Time in there "Main" repository.

Please link your ubuntu tickets here.

Until Ubuntu support BIT I would suggest you to use the latest [release](https://github.com/bit-team/backintime/releases) from our GitHub repo. Currently there is a known Issue about modern rsync versions. It is fixed but not yet released. Because of that you should always use `--old-args` in the extra rsync arguments field in the __Expert Options__ tab. Then everything will be fine.

**Comment by aryoda** [comment]
Currently you can install only from the sources as described here:

https://github.com/bit-team/backintime#from-sources

By default this install the current dev version (`master` branch) which is IMHO more stable than the released v1.3.2
but if you want to install the last release version you can apply `git checkout v.1.3.2` (don't care about the "detached HEAD" msg) and follow the build instructions.

**Comment by jdrch** [comment]
@buhtz @aryoda Thanks for your input. I wound up removing the PPA from my `apt` sources and applying the `--old-args` setting as advised. Tried a backup afterwards and it completed just fine.
